components/LevelDesigner.js

```diff
@@ -0,0 +1,21 @@
+import {useState} from "react";
+import chooseColor from "../functions/chooseColor";
+
+export default function LevelDesigner() {
+  const [array, setArray] = useState(() => {
+    let thing = [];
```

Comment:
This is not a React component and no React hook so it can't use `useState`. Do you need this function at any other place? If not, please delete the file.